src/provisioning/api.md

```diff
@@ -0,0 +1,290 @@
+    }
+}
+```
+
+## DELETE /v1/{roomId}/connections/{id}
+
+Delete a connection. The `id` refers to the `id` returned in the GET response.
+### Response
+
+```json5
+{
+    "ok": true
+}
+```
+
+# Service specific APIs
+
+Some services have specific APIs for additional functionality, like OAuth.
+
+## GitHub
+
+
+### GET /github/v1/oauth?userId={userId}
+
+
+Request an OAuth url for the given user. Once the user has completed the steps in the OAuth process,
+the bridge will be granted access.
+
+### Response
+
+```json5
+[{
+    "url": "https://github.com/login/oauth/authorize?..."
+}]
+```
+
+### GET /github/v1/account?userId={userId}
+
+Request the status of the users account. This will return a `loggedIn` value to determine if the
+bridge has a GitHub identity stored for the user, and any organisations they have access to.
+
+### Response
+
+```json5
+{
+    "loggedIn": true,
+    "organisations": {
+        "name": "half-shot",
+        "avatarUrl": "https://avatars.githubusercontent.com/u/8418310?v=4"
+    }
+}
+```
+
+### GET /github/v1/orgs/{orgName}/repositories?userId={userId}&page={page}&perPage={perPage}
+
+Request a list of all repositories a user is a member of in the given org. The `owner` and `name` value of a repository can be given to create a new GitHub connection.
+
+This request is paginated, and `page` sets the page (defaults to `1`) while `perPage` (defaults to `10`) sets the number of entries per page.
+
+This request can be retried until the number of entries is less than the value of `perPage`.
+
+### Response
+
+```json5
+{
+    "loggedIn": true,
+    "repositories": {
+        "name": "matrix-hookshot",
+        "owner": "half-shot",
+        "fullName": "half-shot/matrix-hookshot",
+        "avatarUrl": "https://avatars.githubusercontent.com/u/8418310?v=4",
+        "description": "A bridge between Matrix and multiple project management services, such as GitHub, GitLab and JIRA. "
+    }
+}
+```
+
+### GET /github/v1/repositories?userId={userId}&page={page}&perPage={perPage}
+
+Request a list of all repositories a user is a member of (including those not belonging to an org). The `owner` and `name` value of a repository can be given to create a new GitHub connection.
+
+This request is paginated, and `page` sets the page (defaults to `1`) while `perPage` (defaults to `10`) sets the number of entries per page.
+
+This request can be retried until the number of entries is less than the value of `perPage`.
+
+### Response
+
+```json5
+{
+    "loggedIn": true,
+    "repositories": {
+        "name": "matrix-hookshot",
+        "owner": "half-shot",
+        "fullName": "half-shot/matrix-hookshot",
+        "avatarUrl": "https://avatars.githubusercontent.com/u/8418310?v=4",
+        "description": "A bridge between Matrix and multiple project management services, such as GitHub, GitLab and JIRA. "
+    }
+}
+```
+
+## JIRA
+
+
+### GET /jira/v1/oauth?userId={userId}
```

Comment:
these endpoints are actually at `/v1/jira`, not `/jira/v1`

likewise for github